x-pack/plugins/observability_solution/infra/public/pages/link_to/redirect_to_host_detail_via_ip.tsx

```diff
@@ -44,13 +64,8 @@ export const RedirectToHostDetailViaIP = ({
     );
   }
 
-  const searchString = replaceMetricTimeInQueryString(
-    getFromFromLocation(location),
-    getToFromLocation(location)
-  )('');
-
   if (name) {
-    return <Redirect to={`/detail/host/${name}?${searchString}`} />;
+    return null;
   }
```

Comment:
Can't you just remove this?

Reply:
We need a return value here and the default is the loading state so put the loading state only when there is no name and set the `null` as default so we still have a default (to avoid any issues and to always have a return value)